Tests/ParseSwiftTests/KeychainStoreTests.swift

```diff
@@ -10,8 +10,13 @@ import Foundation
 import XCTest
 
 @testable import ParseSwift
-
 class KeychainStoreTests: XCTestCase {
+    struct ComplexObject<T: Codable> : Codable {
```

Comment:
This is not what is being tested initially. What was tested was supporting storing arrays of Codables.

Reply:
@flovilmart Thank you. I'm sorry for too many commits in here without considering all test cases. I am going to remove the nested structure.